code/game/gamemodes/meteor/meteors.dm

```diff
@@ -208,7 +213,7 @@ var/global/list/meteors_cataclysm = list(\
 
 /obj/effect/meteor/use_tool(obj/item/tool, mob/user, list/click_params)
 	// Pickaxe - Delete meteor
-	if (istype(tool, /obj/item/pickaxe))
+	if (istype(tool, /obj/item/pickaxe) && !ismissile)
```

Comment:
Why would a pickaxe be a missile? (aside from being dorfy af)

Reply:
ismissile here refers to the meteors, a var Bay added and did almost nothing with

Reply:
Okay, sure, but my point is... why would the tool be a pickaxe AND a meteor? 

If the first istype() passes, I don't see why would the added check ever NOT be True, because it's a pickaxe, not a missile/meteor.

If it didn't pass, the whole check gets short-circuited - there's no point checking further ANDed conditions once the first has failed (similarly, ORing stuff stops at the first True check because the whole thing will be True regardless if anything else fails the check).

So, the `!ismissile` check will either **not run at all**, or **will always pass**. As such, it seems entirely superfluous.

Reply:
Nono, `ismissile` is on the `src` _meteor_ (`!src.ismissile`), NOT the tool. The check makes sense

Reply:
Oh derp. That's why I hate the implicit-src convention =_=

Reply:
yeah, i still use src in a lot of places for my own readability, this might've been one of those places where it would help